GLAA.Domain/GLAAContextExtensions.cs

```diff
@@ -1302,7 +1302,7 @@ private static void AddLicenceWithBusinessDetailsCompleted(this GLAAContext cont
                 var licenceStatusChange = new LicenceStatusChange
                 {
                     DateCreated = DateTime.Now,
-                    Status = context.LicenceStatuses.Find(110)
```

Comment:
Statuses changed so that these applications are no longer "submitted" when seeded.